src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -130,43 +130,6 @@ export default class MutationMapperStore extends DefaultMutationMapperStore
         return mutation.gene.entrezGeneId;
     }
 
-    @autobind
-    protected customFilterApplier(filter: DataFilter,
```

Comment:
This logic has been moved as multiple separate methods into https://github.com/cBioPortal/react-mutation-mapper/blob/66648ae9b966f7a30069e397acfb0e6efc0a0c73/src/store/DefaultMutationMapperFilterApplier.ts#L24